lib/docker-compose-definitions-splunk-helpers.yml

```diff
@@ -0,0 +1,19 @@
+x-tx-gen-def:
+  &tx-gen-def
```

Comment:
tx-gen-def isn't needed with splunk, but is needed with the tx-generator. So this is broken right now if you don't use splunk AND txgenerate. I'd say just don't use a definition here at all and put this directly where you generate the txgen service section.

Reply:
this was removed